scripts/publish.sh

```diff
@@ -539,6 +601,18 @@ if [ "${1}" = "cicd-publish" ]; then
 		sync_latest_image ${bahrain_region} ${bahrain_account_id}
 	elif [ "${2}" = "${hongkong_region}" ]; then
 		sync_latest_image ${hongkong_region} ${hongkong_account_id}
+	elif [ $# -eq 3 ] && [ "${3}" = "stable" ]; then
```

Comment:
In the buildspec I see:

```
- './scripts/publish.sh cicd-publish ${AWS_REGION} stable' 

# Publish stable tag to Dockerhub
- './scripts/publish.sh cicd-publish ${AWS_REGION} public-dockerhub'
```

In both cases, in hong kong, I think this will cause the script to run `sync_latest_image ${hongkong_region} ${hongkong_account_id}`

I think the logic needs to be changed.